requests/__init__.py

```diff
@@ -86,6 +87,25 @@ def check_compatibility(urllib3_version, chardet_version):
 except ImportError:
     pass
 
+# Check for older version of pyOpenSSL
+def check_pyopenssl(cryptography_version, pyopenssl_version):
+    # Only applies to pyOpenSSL
+    if sys.modules['OpenSSL'].__title__ == 'pyOpenSSL':
```

Comment:
I feel like there's a bifurcation. Below we check that `'OpenSSL' in sys.modules` why not do this check there too? Further, is there another `OpenSSL` module that we might import that isn't `pyOpenSSL`? What is this check protecting against?

Reply:
I misread a reply to the #4309 and assumed multiple packages could provide an `OpenSSL` module, but it looks like I was wrong. I'll remove that `__title__` check.